<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBook</class>
 <widget class="QDialog" name="AddBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление новой книги</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="annotation_value">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="title_value">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>300</number>
   </property>
  </widget>
  <widget class="QLabel" name="annotation_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание</string>
   </property>
  </widget>
  <widget class="QLabel" name="genre_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Жанр</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="new_author_rb">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Новый</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="exist_author_rb">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Существующий</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="without_author_rb">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Без автора</string>
   </property>
  </widget>
  <widget class="QLabel" name="author_rb_label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Автор</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="price_value">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100001.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="price_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="amount_value">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1001</number>
   </property>
  </widget>
  <widget class="QLabel" name="amount_label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>270</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fio_author_value">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="fio_autorh_label_line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя автора</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="birthdate_value">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="birthdate_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата рождения</string>
   </property>
  </widget>
  <widget class="QLabel" name="deathdate_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата смерти (опционально)</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="deathdate_value">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="author_names_cb">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="genre_cb">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="is_alive_checkbox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ныне живущий</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_book_button">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="year_publication_value">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="year_publication_label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Год публикации</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
